src/main.py

```diff
@@ -100,6 +100,56 @@ async def get_handler(req):
     return response.json(json_response['results']['bindings'])
 
 
+@app.route('/list', methods=['GET'])
+async def get_handler(req):
+    sparql_url = environ.get('SPARQL_URL')
+    print(check_query_args_list(req.query_args))
+    if not check_query_args_list(req.query_args):
+        return response.json({'message': 'Wrong query parameters!'}, status=400)
```

Comment:
Should also add a `'success': false` in the JSON for consistency (I agreed on doing that with Nawang)